scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+
+from datetime import timedelta
+from faker import Factory
+
+import os
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
```

Comment:
Can we combine `create_user` and `create_admin_user` in a single function. You can take flags for `superuser` and value of username can be kept optional.